src/test/java/gov/usgs/owi/nldi/controllers/RedirectControllerTest.java

```diff
@@ -0,0 +1,36 @@
+package gov.usgs.owi.nldi.controllers;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+import gov.usgs.owi.nldi.services.TestConfigurationService;
+import org.springframework.web.servlet.view.RedirectView;
+
+public class RedirectControllerTest {
+
+
+	private TestConfigurationService configurationService;
+	private RedirectController controller;
+
+	@Before
+	@SuppressWarnings("unchecked")
+	public void setup() {
+		MockitoAnnotations.initMocks(this);
+
+		configurationService = new TestConfigurationService();
+        controller = new RedirectController(configurationService);
+	}
+
+	@Test
+	public void getSwaggerTest() throws Exception {
```

Comment:
This doesn't really check that the redirect (within the app) actually worked. https://github.com/NWQMC/WQP-WQX-Services/blob/master/src/test/java/gov/usgs/wma/wqp/webservice/RedirectControllerTest.java is an example that does.

Reply:
When I copy that test, I'm running into an issue where invoking the controller on "http://localhost:<random>/nldi/swagger" complains that owi-test.usgs.gov is an unknown host.  I see the nldi-services project has owi-test.usgs.gov hardcoded in the TestConfigurationService, but both configurationService.getRootUrl() and restTemplate.getRootUrl() are returning the test url (http://localhost:<random>/nldi) so ... looking in to it.

Reply:
I upgraded the project to JUnit 5 as you suggested and the test looks good now.